org/ystia/terraform/pub/types.yaml

```diff
@@ -30,10 +30,11 @@ node_types:
     tags:
       icon: terraform.jpg
     properties:
-      component_version:
```

Comment:
I never know if we should set this to the default value just as a feedback for the user